### ISSUE TITLE
XML file

### ISSUE THREAD

**Issue by Michael21107** [justalemon/PlayerCompanion#24]
Shouldn't `PlayerCompanion.xml` also be in the `FOR DEVELOPERS` folder?
![image](https://github.com/user-attachments/assets/53c5a26a-3d6f-44a4-9586-5190be458ded)

**Comment by justalemon** [comment]
Nope, as that will be used for automatic menu settings generation during runtime in the future. This requires the xml file to be placed in the same folder.

**Comment by justalemon** [comment]
Ill leave this issue around just in case.

**Comment by Michael21107** [comment]
> Nope, as that will be used for automatic menu settings generation during runtime in the future. This requires the xml file to be placed in the same folder.

cool cool

> Ill leave this issue around just in case.

copy that, thanks
